rogue/src/animation_system.rs

```diff
@@ -242,12 +242,12 @@ fn make_aoe_animation(
     fg: RGB,
     bg: RGB,
     glyph: rltk::FontCharType,
-    radius: i32,
+    radius: f32,
 ) -> Vec<ParticleRequest> {
     let mut particles = Vec::new();
     let fg_color_cycle = [fg, bg, fg];
     let bg_color_cycle = [bg, fg, bg];
-    let mut blast_tiles = rltk::field_of_view(Point { x, y }, radius, &*map);
+    let mut blast_tiles = rltk::field_of_view(Point { x, y }, radius.ceil() as i32, &*map);
```

Comment:
This doesn't quite work. We're constrained here because `field_of_view` expects an integer, so cannot produce a fov within a euclidean disk. We'll fix this in a follow up PR.